qa/L0_client_timeout/test.sh

```diff
@@ -204,11 +238,55 @@ set -e
 kill $SERVER_PID
 wait $SERVER_PID
 
+# Test all APIs other than infer
+export TRITONSERVER_SERVER_DELAY_GRPC_RESPONSE_SEC=2
+SERVER_ARGS="${SERVER_ARGS} --model-control-mode=explicit --load-model=custom_identity_int32 --log-verbose 2"
+run_server
+if [ "$SERVER_PID" == "0" ]; then
+    echo -e "\n***\n*** Failed to start $SERVER\n***"
+    cat $SERVER_LOG
+    exit 1
+fi
+set +e
+
+for i in test_grpc_server_live \
```

Comment:
nit: These looks like too many test cases to iterate through. Can we create a new python test for non-infer API that can be directly invoked from here?
This would allow all the sub-tests to be executed without listing each of them here.